ndarray-rand/Cargo.toml

```diff
@@ -23,6 +23,7 @@ features = ["small_rng"]
 
 [dev-dependencies]
 rand_isaac = "0.2.0"
+quickcheck = { version = "0.8", default-features = false }
```

Comment:
We can upgrade to quickcheck 0.9 I think - but, as a separate task from this PR?